src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
-set(DLC_SRC_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
-set(DLC_BIN_ROOT "${CMAKE_CURRENT_BINARY_DIR}")
+set(NNPA_SRC_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Do we need the repliacated names? e.g. 
```
set(NNPA_ONNX_MLIR_SRC_ROOT ${ONNX_MLIR_SRC_ROOT})
 set(NNPA_ONNX_MLIR_BIN_ROOT ${ONNX_MLIR_BIN_ROOT})
```
I don't think we need to have an `NNPA_ONNX_MLIR_SRC_ROOT` to begin with. 
Let me know  if you prefer having this in a subsequent PR

Reply:
Yes, leave it for another PR to keep this patch self-contained.